scapy/contrib/gtp_v2.py

```diff
@@ -245,9 +245,14 @@ def hashret(self):
         return struct.pack("B", self.version) + self.payload.hashret()
 
     def answers(self, other):
+        # TODO: Also check that the GTP message type of `other` is a response
```

Comment:
Could you implement the logic for GTPV2EchoRequest() and GTPV2EchoResponse(), and the corresponding unit test? That will be a good example to reference in this comment.

Reply:
Do you mean to add a `GTPV2EchoResponse.answers` method?  Or to check that `self.gtp_type` is an answer to `other.gtp_type` in `GTPHeader.answers`?

I don't like the idea of adding a `GTPV2EchoResponse.answers` method:
  * All the information needed to figure out if one GTPv2 message is an answer to another is inside `GTPHeader` already, I don't think it's worth moving it elsewhere.  (There's not enough information to determine non-piggybacked responses to a piggybacked message, but there's no support for piggybacking in the module at all.)
  * Also I'm not quite sure if there's any value in all `GTPV2Command` subclasses.  Right now they don't seem to serve any real purpose, and just add information duplication (between `GTPmessageType` dict, the `GTPV2Command` subclasses, and `bind_layers(*)` statements).  I'd rather remove all the `GTPV2Command` subclasses, or at least somehow reduce the duplication. 

I feel that adding logic to `GTPHeader.answers` to handle *some* response/request is a good idea.  I'd prefer it to be either complete (having logic for all the message types from the spec), or obviously basic.  It seems to me that it that the behavior of the module will be less surprising that way.

On the other hand I don't really care too much about implementation as long as my `sr1` is working :).  So, what option from above you'd like me to implement?